src/core/generators/mocks.ts

```diff
@@ -84,18 +86,27 @@ const getMockWithoutFunc = (
     : {}),
 });
 
-const getMockScalarJsTypes = (definition: string) => {
+const getMockScalarJsTypes = (
+  definition: string,
+  mockOptionsWithoutFunc: { [key: string]: unknown },
+) => {
   const isArray = definition.endsWith('[]');
   const type = isArray ? definition.slice(0, -2) : definition;
 
   switch (type) {
     case 'number':
       return isArray
-        ? `Array.from({length: faker.datatype.number({min: 1, max: 10})}, () => faker.datatype.number())`
+        ? `Array.from({length: faker.datatype.number({` +
+            `min: ${mockOptionsWithoutFunc?.arrayMin ?? 1}, ` +
+            `max: ${mockOptionsWithoutFunc?.arrayMax ?? 10}}` +
+            `)}, () => faker.datatype.number())`
         : 'faker.datatype.number().toString()';
     case 'string':
       return isArray
-        ? `Array.from({length: faker.datatype.number({min: 1, max: 10})}, () => faker.random.word())`
+        ? `Array.from({length: faker.datatype.number({` +
+            `min: ${mockOptionsWithoutFunc?.arrayMin ?? 1},` +
```

Comment:
can you put the default values in the normalizer of the options?

Reply:
Fixed.